store/rootmulti/store_test.go

```diff
@@ -82,7 +82,7 @@ func TestCacheMultiStoreWithVersion(t *testing.T) {
 	store1 := ms.GetStoreByName("store1").(types.KVStore)
```

Comment:
is it possible / easy to add a unit test for the case where we use `Commit(false)`? if not it's fine

Reply:
good point. Added one